plasma_framework/test/endToEndTests/PaymentV2Experiment.e2e.test.js

```diff
@@ -0,0 +1,146 @@
+const EthVault = artifacts.require('EthVault');
+const Erc20Vault = artifacts.require('Erc20Vault');
+const ExitableTimestamp = artifacts.require('ExitableTimestampWrapper');
+const ExitPriority = artifacts.require('ExitPriorityWrapper');
+const ERC20Mintable = artifacts.require('ERC20Mintable');
+const PaymentExitGame = artifacts.require('PaymentExitGame');
+const PlasmaFramework = artifacts.require('PlasmaFramework');
+
+const {
+    BN, constants, time,
+} = require('openzeppelin-test-helpers');
+const { expect } = require('chai');
+
+const { MerkleTree } = require('../helpers/merkle.js');
+const { PaymentTransactionOutput, PaymentTransaction } = require('../helpers/transaction.js');
+const { buildUtxoPos } = require('../helpers/positions.js');
+const Testlang = require('../helpers/testlang.js');
+const config = require('../../config.js');
+
+contract('PaymentExitGame - V2 Extension experiment', ([_deployer, _maintainer, authority, richFather]) => {
+    const ETH = constants.ZERO_ADDRESS;
+    const INITIAL_ERC20_SUPPLY = 10000000000;
+    const DEPOSIT_VALUE = 1000000;
+    const OUTPUT_TYPE_PAYMENT = config.registerKeys.outputTypes.payment;
+    const OUTPUT_TYPE_PAYMENT_V2 = config.experimental.registerKeys.outputTypes.paymentV2;
+    const PAYMENT_V2_TX_TYPE = config.registerKeys.txTypes.paymentV2;
+    const MERKLE_TREE_DEPTH = 16;
+
+    const alicePrivateKey = '0x7151e5dab6f8e95b5436515b83f423c4df64fe4c6149f864daa209b26adb10ca';
```

Comment:
Can we load the private key from the the environment?

Reply:
we could but why?

Reply:
It's generally a bad practice to have private keys in code.